images/linux/scripts/SoftwareReport/SoftwareReport.Generator.ps1

```diff
@@ -246,16 +246,11 @@ $browsersAndDriversList = @(
     (Get-ChromiumVersion),
     (Get-EdgeVersion),
     (Get-EdgeDriverVersion),
-    (Get-SeleniumVersion)
+    (Get-SeleniumVersion),
+    (Get-FirefoxVersion),
+    (Get-GeckodriverVersion)
```

Comment:
We should update `ubuntu2204.pkr.hcl` file to include firefox.sh script.

Reply:
The file was updated.